pylsp/workspace.py

```diff
@@ -14,6 +14,8 @@
 
 log = logging.getLogger(__name__)
 
+DEFAULT_AUTO_IMPORT_MODULES = ["numpy", "gi"]
```

Comment:
```suggestion
DEFAULT_AUTO_IMPORT_MODULES = ["numpy"]
```

I don't think `gi` should be here either.